From 9bd468920e23649e1244c5d243215c04ff3e72e6 Mon Sep 17 00:00:00 2001
From: Viorel Suman <viorel.suman@nxp.com>
Date: Tue, 18 Jun 2019 16:16:19 +0300
Subject: [PATCH] MLK-21957-1: ASoC: fsl_sai: remove reset code from dai_probe

SAI software reset is done in runtime resume,
there is no need to do it in fsl_sai_dai_probe.

Signed-off-by: Viorel Suman <viorel.suman@nxp.com>
Signed-off-by: Shengjiu Wang <shengjiu.wang@nxp.com>
---
 sound/soc/fsl/fsl_sai.c | 7 -------
 1 file changed, 7 deletions(-)

--- a/sound/soc/fsl/fsl_sai.c
+++ b/sound/soc/fsl/fsl_sai.c
@@ -991,13 +991,6 @@ static int fsl_sai_dai_probe(struct snd_
 	struct fsl_sai *sai = dev_get_drvdata(cpu_dai->dev);
 	unsigned char offset = sai->soc->reg_offset;
 
-	/* Software Reset for both Tx and Rx */
-	regmap_write(sai->regmap, FSL_SAI_TCSR(offset), FSL_SAI_CSR_SR);
-	regmap_write(sai->regmap, FSL_SAI_RCSR(offset), FSL_SAI_CSR_SR);
-	/* Clear SR bit to finish the reset */
-	regmap_write(sai->regmap, FSL_SAI_TCSR(offset), 0);
-	regmap_write(sai->regmap, FSL_SAI_RCSR(offset), 0);
-
 	regmap_update_bits(sai->regmap, FSL_SAI_TCR1(offset),
 				sai->soc->fifo_depth - 1,
 				sai->soc->fifo_depth - FSL_SAI_MAXBURST_TX);
